app/src/main/java/com/example/news_app/VideoAdapter.java

```diff
@@ -23,19 +23,31 @@ public class VideoAdapter extends RecyclerView.Adapter<VideoAdapter.ViewHolder>
 
     private final Context context;
     private final List<Video> videos;
+    private final ViewType viewType;
 
 
-    public VideoAdapter(Context context, List<Video> videos) {
+    public VideoAdapter(Context context, List<Video> videos, ViewType viewType) {
         this.context = context;
         this.videos = videos;
+        this.viewType= viewType;
     }
 
-
     @NonNull
     @Override
     public ViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int viewType) {
+        View videoView = null;
+        switch (this.viewType) {
+            case OneColumn:
```

Comment:
naming convention for enum values should be all caps
https://stackoverflow.com/a/3069863